tests/augment/test_keboard_typo.py

```diff
@@ -0,0 +1,20 @@
+import random
+from typing import List
+
+import pytest
+
+from langcheck.augment import keyboard_typo
+
+
+@pytest.mark.parametrize(
+    "texts, expected",
+    [
+        (["Hello, world!"], ["HF;lo, eorlE!"]),
```

Comment:
The perturbation seems too much as the default setting to me. How about using some values like `aug_char_p=0.1` as the default?

```python
>>> import nlpaug.augmenter.char as nac
>>> import random
>>> random.seed(42)
>>> nac.KeyboardAug().augment(['Hello, world!!'])
['HF;lo, eorlE!!']
>>> nac.KeyboardAug(aug_char_p=0.1).augment(['Hello, world!!'])
['He?lo, w*rld!!']
```

Reply:
`include_special_char=False, include_numeric=False` should be reasonable as the default setting too